cla_public/apps/base/views.py

```diff
@@ -154,6 +154,7 @@ def show_session():
 
 @base.route("/session-expired")
 def session_expired():
+    session.clear()
```

Comment:
Was it a mistake that this wasn't there before, or is it newly required because we're redirecting from the front end now?

Reply:
We are redirecting to it from the front-end now. So we added have a 5 minutes countdown and if the user doesn't cancel they're redirected to this page.

I think this page should end the session if hasn't already as it's showing the session-expired page to the user.

Reply:
Good point